packages/varnish/src/components/header/AI2Banner.tsx

```diff
@@ -1,10 +1,14 @@
 import * as React from 'react';
 import styled from 'styled-components';
-import { AI2Logo } from '../logos/AI2Logo';
+import Grid from '@mui/material/Grid';
+import { AI2Logo, LogoSize } from '../logos/AI2Logo';
 import { Content } from '../Content';
 
 export interface AI2BannerProps {
   children?: React.ReactNode;
+  size?: LogoSize;
+  controls?: React.ReactNode;
```

Comment:
allows us to add buttons or controls to the right side of the banner, as in the OLMo Figma: 

<img width="1241" alt="Screen Shot 2023-07-12 at 4 18 43 PM" src="https://github.com/allenai/varnish-mui/assets/6945628/ec6102dc-77ce-4bc9-b1c4-025cb732eaf0">